scripts/release-notes.txt

```diff
@@ -1,5 +1,14 @@
 Release Notes of the ESBMC model checker
 
+*** Version 6.10
+* Lots of bugs cases relating unions have been fixed
```

Comment:
english b0rked. "Lots of bugs related to unions have been fixed".

Reply:
Done